### PR TITLE
[helm] Update deprecation notice

### PR DESCRIPTION
**Pull request by vsliouniaev**
There are still some folks coming to the kubernetes slack and asking for information on how to use the chart from this repository.

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -1,37 +1,15 @@
...
+# THIS CHART IS DEPRECATED
+The chart has been replaced by [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator) and is no longer maintained. Therefore we are currently not accepting any pull requests. Thank you for your understanding.
 
-You can track the progess of the work [here](https://github.com/coreos/prometheus-operator/issues/592) and [here](https://github.com/helm/charts/pull/6765)
+You can check out the tickets for this change [here](https://github.com/coreos/prometheus-operator/issues/592) and [here](https://github.com/helm/charts/pull/6765)
 
-# TL;DR
+---
+
+#### How to install this chart
```
Comment (brancz) [inline helm/README.md]: Can we maybe delete this along with the rest of the files?

**Comment by brancz** [comment]
As @gianrubio seems no longer active and you two @paskal and @vsliouniaev seem to have largely taken over, how do you two feel about just removing the files here entirely and just keeping the redirect readme?
